README.md

```diff
@@ -1,6 +1,6 @@
 # UpCloud Go API client library
 
-[![Build Status](https://travis-ci.org/UpCloudLtd/upcloud-go-api.svg?branch=master)](https://travis-ci.org/UpCloudLtd/upcloud-go-api)
+![Build Status](https://github.com//UpCloudLtd/upcloud-go-api/workflows/Upcloud%20go%20api%20test/badge.svg)
```

Comment:
I'm guessing this is a typo `https://github.com//UpCloudLtd`and should be with one /

Reply:
Also this change is missing from the Python api. So a good catch 😄